src/Gui/FreeCADGuiInit.py

```diff
@@ -124,6 +126,27 @@ def GetClassName(self):
         """Return the name of the associated C++ class."""
         return "Gui::NoneWorkbench"
 
+InputSequence = Union[Gui.UserInput, tuple[Gui.UserInput, ...]]
+
+@dataclass
+class InputHint:
+    message: str
+    sequences: list[InputSequence]
+
+    def __init__(self, message: str, *sequences: InputSequence):

```

Comment:
We could put the InputSequence here instead as it is only used inside the class.

```python
@dataclass
class InputHint:
    InputSequence = Gui.UserInput | tuple[Gui.UserInput, ...]  # <---

    message: str
    sequences: list[InputSequence]

    def __init__(self, message: str, *sequences: InputSequence):
```